site/pages/what-we-do/cross-functional-slice/index.js

```diff
@@ -0,0 +1,73 @@
+// @flow
+import React from 'react';
+import styles from './styles.css';
+import ColourBox from '../../../components/colour-box';
+import Image from '../../../components/image';
+
+import bigParty1x from './png/big-party.png';
+import bigParty2x from './png/big-party@2x.png';
+import bigParty3x from './png/big-party@3x.png';
+import crossFunctionalBadgers from './jpg/cross-functional-badgers.jpg';
+
+const content = [
+  {
+    title: 'Lean Agile',
+    labelColour: 'blue',
+    description: 'Drive efficiency and reduce risk through our lean methods.',
+  },
+  {
+    title: 'Tech',
+    labelColour: 'green',
+    link: 'technology',
+    description: 'Transform your business by being bold with tech.',
+  },
+  {
+    title: 'Design',
+    labelColour: 'yellow',
+    description: 'Improve customer experience, create delightful products and services.',
+  },
+  {
+    title: 'You (the client)',
+    labelColour: 'red',
+    description: 'Together we build a capability and lasting change.',
+  },
+];
+
+const CrossFunctionalSlice = () => (
+  <section className={styles.crossFunctionalSlice}>
+    <div className={styles.photoContainer}>
+      <Image
+        src={crossFunctionalBadgers}
+        alt="working in a cross functional team"
```

Comment:
Think we should improve this alt text to be more descriptive

Reply:
I have changed this to 'a photo showing Red Badger employees working in a cross functional team'